### PR TITLE
Mention that `int()` can be used as an alternative to `floor()`

### PR DESCRIPTION
**Pull request by Calinou**
See discussion in #19262.

### CODE REVIEW

File: modules/gdscript/doc_classes/@GDScript.xml [hunk 1]
```diff
@@ -341,6 +341,7 @@
 				# a is -3
 				a = floor(-2.99)
 				[/codeblock]
+				[b]Note:[/b] This method returns a float. If you need an integer, you can use [code]int(s)[/code] directly.
```
Comment (akien-mga) [inline modules/gdscript/doc_classes/@GDScript.xml]: Maybe also change the codeblock so that the comments use `2.0` and `-3.0`?

Comment (Calinou) [reply modules/gdscript/doc_classes/@GDScript.xml]: @akien-mga Good catch :slightly_smiling_face:

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by vnen** [comment]
This can be a bit misleading though, because `floor(-2.99)` is `-3` but `int(-2.99)` is `-2`.

**Pull request merged**